solr/solrj/src/java/org/apache/solr/common/util/ExecutorUtil.java

```diff
@@ -83,6 +83,24 @@ public interface InheritableThreadLocalProvider {
     void clean(AtomicReference<Object> ctx);
   }
 
+  public static boolean isShutdown(ExecutorService pool) {
+    try {
+      return pool.isShutdown();
+    } catch (IllegalStateException e) {
+      // JSR-239 ManagedExecutorService cannot query the lifecycle, so just return false
```

Comment:
❌ JSR-239
⭕️ JSR-236

Reply:
thanks fixing in a few minutes.

Reply:
Fixed in 8736c991bb6b80c11de06fbcb499ce8dbe2a7cb2